src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/AssemblyNameInfoFuzzer.cs

```diff
@@ -26,19 +30,51 @@ public void FuzzTarget(ReadOnlySpan<byte> bytes)
 
         private static void Test(PooledBoundedMemory<char> inputPoisoned)
```

Comment:
NIT: could change the parameter to span as only span of the poisoned input is used in the method
```suggestion
        private static void Test(Span<char> span)
```

Reply:
> NIT: could change the parameter to span as only span of the poisoned input is used in the method

If that is not a problem I am going to merge it as is and include this change in next PR.